zenml/core/backends/orchestrator/aws/orchestrator_aws_backend.py

```diff
@@ -0,0 +1,144 @@
+#  Copyright (c) maiot GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Orchestrator for simple AWS VM backend"""
+
+import os
+import time
+from typing import Text, Dict, Any
+
+from zenml.core.backends.orchestrator.aws import utils
+from zenml.core.backends.orchestrator.base.orchestrator_base_backend import \
+    OrchestratorBaseBackend
+from zenml.core.repo.repo import Repository
+from zenml.core.standards import standard_keys as keys
+from zenml.utils import path_utils
+from zenml.utils.constants import ZENML_BASE_IMAGE_NAME
+from zenml.utils.logger import get_logger
+
+logger = get_logger(__name__)
+
+EXTRACTED_TAR_DIR_NAME = 'zenml_working'
+STAGING_AREA = 'staging'
+TAR_PATH_ARG = 'tar_path'
+
+
+class OrchestratorAWSBackend(OrchestratorBaseBackend):
+    """
+    Orchestrates pipeline on a AWS EC2 instance
+    """
+
+    def __init__(self,
+                 iam_role: Text,
+                 instance_type: Text = 't2.micro',
+                 instance_image: Text = 'ami-02e9f4e447e4cda79',
```

Comment:
Can you add a comment here what this instance image is? Ubuntu?

Reply:
Added a docstring to make it more comprehensive!